rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ]  Let's add `:book` and `:person` to attr_accessor as users can write and read them. See suggestion below:

```suggestion
  attr_accessor :date, :person, :book
```